dissect/esedb/record.py

```diff
@@ -217,7 +260,11 @@ def _parse_value(self, column: Column, value: bytes, tag_field: TagField = None)
             if tag_field and tag_field.flags & TAGFLD_HEADER.MultiValues:
                 value = list(map(parse_func, value))
             else:
-                value = parse_func(value)
+                try:
+                    value = parse_func(value)
+                except UnicodeDecodeError as e:
+                    # at least return the raw value
+                    raise RuntimeError(f"{hexlify(bytes(value)).decode()} [{type(e).__name__}: {e}]")
```

Comment:
```suggestion
                value = parse_func(value)
```
I don't think this is the correct place to do this. Do you have cases that consistently break?

Reply:
Indeed, the raw value can be retrieved elsewhere in case of an error. We stumbled upon decoding errors on some attributes in an NTDS.